src/main/scala/com/nec/spark/planning/ParallelCompilationColumnarRule.scala

```diff
@@ -1,66 +1,65 @@
-package com.nec.spark.planning
+// package com.nec.spark.planning
```

Comment:
I guess the file should be deleted before this is merged.

Reply:
Done